jinja2cli/cli.py

```diff
@@ -220,7 +220,13 @@ def _parse_env(data):
 }
 
 
-def render(template_path, data, extensions, strict=False):
+def _load_ansible_filters():
+    from ansible.plugins.filter.core import FilterModule
+
+    return FilterModule().filters()
```

Comment:
What if we `try...except ImportError` and print a more useful message to say that `ansible-core` is missing? Just so we can suggest how to resolve the issue.